app/scripts/templates/settings/account_recovery/recovery_key.mustache

```diff
@@ -0,0 +1,33 @@
+<div id="account-recovery-key">
+    <section class="modal-panel">
+        <div class="error"></div>
+
+        <h2>{{#t}}Recovery key{{/t}}</h2>
+
+        <p class="description">
+            <span class="note">{{#t}}NOTE:{{/t}}</span>
+            {{#t}}Keep your recovery key in a safe place where you can access it later. You will not be able to access this page again. You can download or print your key or email it to someone you trust.{{/t}}
+        </p>
+
+        <div class="recovery-key-container">
+            <div class="recovery-key">{{recoveryKey}}</div>
+        </div>
+
+        <div class="options">
+            <div class="title-container">
+                <div class="line">&boxh;&boxh;&boxh;&nbsp;</div>
+                <div class="title">{{#t}}Download your recovery key or print it{{/t}}</div>
+                <div class="line">&nbsp;&boxh;&boxh;&boxh;</div>
+            </div>
+
+            <div class="buttons">
+                <button class="download-key"><img class="recovery-icon-option" src="/images/recovery_key_download.svg">{{#t}}Download{{/t}}</button>
```

Comment:
Neither of these buttons have styles, is that expected?

Reply:
The styles get putted in [here](https://github.com/mozilla/fxa-content-server/blob/823d5983732db81ee669a3ea8eaa09fc125f3620/app/styles/modules/_settings-account-recovery.scss#L113). I could give them an explicit class though.

Reply:
>> Neither of these buttons have styles, is that expected?

> I could give them an explicit class though.

Poor choice of wording on my part. I meant the buttons are both white with a grey outline, is that the style you are going for?

Reply:
Ah, yea that is the style I am going for.